subsys/bluetooth/host/l2cap.c

```diff
@@ -1419,6 +1424,8 @@ static void le_ecred_reconf_rsp(struct bt_l2cap *l2cap, uint8_t ident,
 			ch->chan.ops->reconfigured(&ch->chan);
 		}
 	}
+
+	BT_DBG("");
```

Comment:
Remove?

Reply:
I added this to get the function name in the log, making sure we process the response. I can remove it if you really want.